pkg/scheduler/plugins/drf/drf.go

```diff
@@ -197,6 +197,38 @@ func (drf *drfPlugin) compareQueues(root *hierarchicalNode, lqueue *api.QueueInf
 	return 0
 }
 
+func (drf *drfPlugin) scoreQueues(root *hierarchicalNode, lqueue *api.QueueInfo, rqueue *api.QueueInfo) (float64, float64) {
```

Comment:
The meaning of the two returns adds some comments information, can it be easier to understand?

Reply:
Done.